CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/value/types/booleanType/BooleanLogicConverters.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.graph.value.types.booleanType;
```

Comment:
So put all the types directly into the types directory? It would contain around 50 classes if I'm understanding correctly

Reply:
Splitting in packages by type isn't really something that has been done in the rest of the code base which is why I made the suggestion. As I say it out loud though, 50 classes does seem like a lot to put into one package. Three options I can think of for this:
1) Leave the structure as is and alter the names of sub-directories of types from camel case to lower case (e.g. booleanType -> booleantype) to follow recommended naming convention
2) Move all the files from sub-directories of types directly into the types directory
3) Restructure the sub-directories of types to be split by functionality rather than by type (logic converters, value stores, etc)

Reply:
I'm liking the sound of 3, it sounds like it fits with what is currently throughout the project.
Just on that, @sirius-1 I was wondering the difference of IntValue within value.values and the IntValue within Value.types.integerType.

Reply:
Yeah, that is a pretty bad name clash. integerType.IntValue is actually the type used extensively in the framework. values.IntValue was just an implementation designed to be used programmatically to allow the insertion of values into expressions when needed. Looking at them now, the integerType.IntValue class does seem to provide all the functionality of the other and make the values.IntValue redundant. It looks like they are not used anyway so maybe just delete?